probe.go

```diff
@@ -801,6 +801,140 @@ func probeLinkMonitor(c FortiHTTP) ([]prometheus.Metric, bool) {
 	return m, true
 }
 
+func probeVirtualWanPerf(c FortiHTTP) ([]prometheus.Metric, bool) {
+	var (
+		mLink = prometheus.NewDesc(
+			"fortigate_virtualwan_healthcheck_status",
+			"Whether the link is up or not",
```

Comment:
In the current form it shows if the specific `state` is active and not directly if it is up or down.